CodeSniffer/Standards/Generic/Sniffs/ControlStructures/InlineControlStructureSniff.php

```diff
@@ -171,6 +171,10 @@ public function process(PHP_CodeSniffer_File $phpcsFile, $stackPtr)
                 if ($endLine !== $end) {
                     $phpcsFile->fixer->addContent($endLine, '}');
                 } else {
+                    if (in_array($tokens[$end]['code'], array(T_SEMICOLON, T_CLOSE_CURLY_BRACKET)) === false) {
```

Comment:
If there are, I couldn't come up with them.  Either way it's an improvement over it's current state.